hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -155,12 +166,20 @@ public Operation convert(String value) throws ParameterException {
         required = true)
     public String targetBasePath;
 
+    @Parameter(names = {"--base-path-prefix"},
```

Comment:
Yes, this is needed for HoodieMultiTableDeltaStreamer. Now I am defining a separate config class for MultiTableStreamer. All unused configs can be removed from HoodieDeltaStreamer.Config class.

Reply:
@pratyakshsharma : Instead of reusing HoodieDeltaStreamer.Config, Can you introduce an inner class HoodieMultiTableDeltaStreamer.Config and have all these configs  (duplicated) along with basePathPrefix. When we instantiate HoodieDeltaStreamer instance in HoodieMultiTableDeltaStreamer, we can create new instance of HoodieDeltaStreamer.Config  by copying all the parameters from HoodieMultiTableDeltaStreamer.Config. This will avoid introducing a config parameter in HoodieDeltaStreamer.Config which has only relevance in HoodieMultiTableDeltaStreamer and keep the CLI parameters for both of them decoupled.